[Dashboard] Total in out card enhancement

mmioana opened the pull request:
## Description

- This PR addresses the `Payments` and `Income` sections from the `Total in and out` card

![Screenshot 2024-09-10 at 10 51 10](https://github.com/user-attachments/assets/51dece5a-2cd4-4630-b0dd-ce9f12332e7e)


## Testing

TODO: Please check the payments and income sections are according to [Figma](https://www.figma.com/design/gac6xVgGNCAge5Bia933dp/Dashboard?node-id=4878-89693&node-type=frame&t=xND2OvegDeZWkrKJ-0)

* Step 1. Run `create-data` script
* Step 2. Go to `planex` dashboard page http://localhost:9091/planex/
* Step 3. Check the Payments and Income values are the same as the ones displayed in the chart for the current month
* Step 4. Create a `Mint tokens` action/motion
* Step 5. After a bit, the income value should update
* Step 6. In order to not directly claim the minted tokens, please create a patch file with the following content - you can name it `.diff`
```
diff --git a/src/redux/sagas/actions/mintTokens.ts b/src/redux/sagas/actions/mintTokens.ts
index d13052bee..12c7658e7 100644
--- a/src/redux/sagas/actions/mintTokens.ts
+++ b/src/redux/sagas/actions/mintTokens.ts
@@ -62,18 +62,18 @@ function* createMintTokensAction({
       ready: false,
     });
 
-    yield fork(createTransaction, claimColonyFunds.id, {
-      context: ClientType.ColonyClient,
-      methodName: 'claimColonyFunds',
-      identifier: colonyAddress,
-      params: [nativeTokenAddress],
-      group: {
-        key: batchKey,
-        id: metaId,
-        index: 1,
-      },
-      ready: false,
-    });
+    // yield fork(createTransaction, claimColonyFunds.id, {
+    //   context: ClientType.ColonyClient,
+    //   methodName: 'claimColonyFunds',
+    //   identifier: colonyAddress,
+    //   params: [nativeTokenAddress],
+    //   group: {
+    //     key: batchKey,
+    //     id: metaId,
+    //     index: 1,
+    //   },
+    //   ready: false,
+    // });
 
     if (annotationMessage) {
       yield fork(createTransaction, annotateMintTokens.id, {
@@ -91,7 +91,7 @@ function* createMintTokensAction({
     }
 
     yield takeFrom(mintTokens.channel, ActionTypes.TRANSACTION_CREATED);
-    yield takeFrom(claimColonyFunds.channel, ActionTypes.TRANSACTION_CREATED);
+    // yield takeFrom(claimColonyFunds.channel, ActionTypes.TRANSACTION_CREATED);
 
     if (annotationMessage) {
       yield takeFrom(
@@ -108,9 +108,9 @@ function* createMintTokensAction({
       },
     } = yield waitForTxResult(mintTokens.channel);
 
-    yield initiateTransaction(claimColonyFunds.id);
+    // yield initiateTransaction(claimColonyFunds.id);
 
-    yield waitForTxResult(claimColonyFunds.channel);
+    // yield waitForTxResult(claimColonyFunds.channel);
 
     yield createActionMetadataInDB(txHash, customActionTitle);
 

```
* Step 7. Apply the patch running `git apply <patch_filename>`
* Step 8. Create another `Mint tokens` action/motion
* Step 9. Check the Claim button appears in the `Income` section

![Screenshot 2024-09-10 at 11 13 24](https://github.com/user-attachments/assets/93f781a1-d011-4246-b71d-5f4bb504b0a1)

* Step 10. Claim the pending claimable funds and check the `Funds claimed` button appears

![Screenshot 2024-09-10 at 11 14 59](https://github.com/user-attachments/assets/56d8fff2-da9a-42d7-b02a-46a7110727ad)

* Step 11. Click the `Add` button and check the Add funds modal appears

![Screenshot 2024-09-10 at 11 16 41](https://github.com/user-attachments/assets/a0593bbc-3ad0-4c51-82ed-94a4ee7fa71f)

* Step 12. Click the `Pay` button and check the `Simple payment` action sidebar is opened with the current team pre-filled

## Diffs

**New stuff** ✨

* The `Payments` and `Income` sections for the `Total in and out` card

**Deletions** ⚰️

* `BalanceModal` was discarded in favour of  `AddFundsModal`

Resolves #2853 #2854 #2855 #2856

Nortsova reviewed (approved):
Values are the same for the chart and title: 
<img width="709" alt="image" src="https://github.com/user-attachments/assets/36ce61f7-5eef-432e-8a60-8a47ec958953">

After minting the token and refreshing, the income value is updated: 
<img width="750" alt="image" src="https://github.com/user-attachments/assets/9c5d4b1f-696c-4f57-89e6-14c47e70d13a">

After the patch and minting the token, I can see the claim button:
<img width="693" alt="image" src="https://github.com/user-attachments/assets/ec6e2706-b958-4825-9088-882a2a64a53c">

Funds successfully claimed:
![Sep-10-2024 21-02-58](https://github.com/user-attachments/assets/34c36074-eb2b-42af-8327-397b6b036049)

Clicking the add button for funds: 
<img width="1510" alt="image" src="https://github.com/user-attachments/assets/831402df-3e27-4cf3-ba22-cd1fb810bb4e">

Note: Shouldn't everything be blurred? Right now, it looks like only the header is blurred.

Also, after paying someone, the "Payments" value hasn't changed.

rdig commented on src/utils/teams.ts:
```diff
@@ -78,6 +78,45 @@ export const getTeamHexColor = (color?: DomainColor) => {
   }
 };
 
+export const getTeamHexSecondaryColor = (color?: DomainColor) => {
```
:+1:

rdig reviewed (changes requested):
Mostly good, well... all good, except a bit of UI nonsense that happens with larger amounts, that's the only reason I've rejected this.

Loading:

![Screenshot from 2024-09-12 09-23-54](https://github.com/user-attachments/assets/81fd062f-25fa-4f94-9d8e-f8d3b1684341)

Initial:

![Screenshot from 2024-09-12 09-23-57](https://github.com/user-attachments/assets/c8f7b9ba-58a0-4e61-b623-e614d0ad14ac)

Mint with claim:

![Screenshot from 2024-09-12 09-25-57](https://github.com/user-attachments/assets/e057644b-63cb-4ded-b436-30373e1ba652)

Updates after mint:

![Screenshot from 2024-09-12 09-26-08](https://github.com/user-attachments/assets/32a51a99-c1aa-4adf-acfb-18e7aaaf4d4a)

Mint without claim:

![Screenshot from 2024-09-12 09-33-39](https://github.com/user-attachments/assets/00e8e3fb-414c-4f32-a56a-fcc4998d383d)

Claiming:

![Screenshot from 2024-09-12 09-33-59](https://github.com/user-attachments/assets/4898c775-25c0-45da-b760-a01e6040de90)
![Screenshot from 2024-09-12 09-34-25](https://github.com/user-attachments/assets/8ac8adca-e453-4edb-b42e-1720526a6df0)

Deposit Modal:

![Screenshot from 2024-09-12 09-34-51](https://github.com/user-attachments/assets/c18b443c-77d1-46c5-8488-36a4d3090fff)

Simple Payment pre-filled:

![Screenshot from 2024-09-12 09-35-19](https://github.com/user-attachments/assets/57f359f9-eb93-4262-a121-bab7d31f6b2a)

As you can see when "Funds claimed" success message shows up, if the amount is large enoung _(well... long enough anyway)_, it will push the button(s) the next row, causing the UI to break slightly.

![Screenshot from 2024-09-12 09-34-25](https://github.com/user-attachments/assets/8ac8adca-e453-4edb-b42e-1720526a6df0)

As a side note, and not related to this PR necessarily, I couldn't manage to apply your patch manually:

![Screenshot from 2024-09-12 09-28-53](https://github.com/user-attachments/assets/71494076-e811-4abe-9c00-c7884c2524cb)

Which is super weird, since I made the changes myself, by hand, and then compared your patch and my changes, and, besides one empty new line, they're the same.

The empty line can't really be the cause since that is removed by default automatically anyway when you save the file.

![Screenshot from 2024-09-12 09-32-33](https://github.com/user-attachments/assets/a20a56ce-4579-435c-afb6-075e56a1974f)

iamsamgibbs commented:
> Regarding the modal background blur, I didn't change any behaviour regarding it, so if @melyndav considers this to be needed, we can create a separate issue for it 🌞

@mmioana Just a note to say this is blurred on master so something has broken along the way (If the issue hasn't been introduced in this PR, then I suspect this is a problem on the navigation branch, but I haven't checked it there, it probably makes more sense to fix it there if that is the case)
 
<img width="1728" alt="Screenshot 2024-09-12 at 09 10 59" src="https://github.com/user-attachments/assets/1cb04c5e-33ff-4d89-9678-e8351949418d">

mmioana commented:
> Mostly good, well... all good, except a bit of UI nonsense that happens with larger amounts, that's the only reason I've rejected this.
> 
> Loading:
> 
> ![Screenshot from 2024-09-12 09-23-54](https://private-user-images.githubusercontent.com/1193222/366769315-81fd062f-25fa-4f94-9d8e-f8d3b1684341.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTMxNS04MWZkMDYyZi0yNWZhLTRmOTQtOWQ4ZS1mOGQzYjE2ODQzNDEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YTkwNjQyMDJlYzkwMmU0OWUwNjU0NDgxMjY5ODBlNmUyODRjZTBhOTdiYmQ2NjI1ZjYyNDNjNzhjMTc3MjVlNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GhyoyNW_-EdkCm1fjg7yzr5oYNDaAfFmn5_xrAOogK0)
> 
> Initial:
> 
> ![Screenshot from 2024-09-12 09-23-57](https://private-user-images.githubusercontent.com/1193222/366769324-c8f7b9ba-58a0-4e61-b623-e614d0ad14ac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTMyNC1jOGY3YjliYS01OGEwLTRlNjEtYjYyMy1lNjE0ZDBhZDE0YWMucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODBiNmNlZWQwZmZmOTFhNjE0YTgxZDRjM2MwZWY1ZDBlYWY2ZmFlN2RmNzE3MjM1MDI5YWI3Zjg3NTRlN2Q5NSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.GWmbag930p0U2aXxa3pW6wXAMJ_V3YxM_lawmjkkHek)
> 
> Mint with claim:
> 
> ![Screenshot from 2024-09-12 09-25-57](https://private-user-images.githubusercontent.com/1193222/366769333-e057644b-63cb-4ded-b436-30373e1ba652.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTMzMy1lMDU3NjQ0Yi02M2NiLTRkZWQtYjQzNi0zMDM3M2UxYmE2NTIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDI2YzU1YWQxZmVkOWY2NDUwNTIyNmM4N2Y0MzllYjQ0ZDIwODJlMDUwYTBlNWM1YTE0ODE5NWUzZTg1YWViYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.37bqBWDYQtGtNcperaD6EY8RiJWmZTd8ZzzdVXyQtmA)
> 
> Updates after mint:
> 
> ![Screenshot from 2024-09-12 09-26-08](https://private-user-images.githubusercontent.com/1193222/366769340-32a51a99-c1aa-4adf-acfb-18e7aaaf4d4a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM0MC0zMmE1MWE5OS1jMWFhLTRhZGYtYWNmYi0xOGU3YWFhZjRkNGEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmIxZTQxYzkxODNkOGMzNzJmZDNmYTgzZGFlMTk5Y2UxZjUzM2VlMDJjYTkyYzFjYmJkMGQ3MTExOWJjYTQ0MCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.zE3zSsFEb-W32kDuZe-TjIw5xbeFzeXEzdI2XWvZQis)
> 
> Mint without claim:
> 
> ![Screenshot from 2024-09-12 09-33-39](https://private-user-images.githubusercontent.com/1193222/366769356-00e8e3fb-414c-4f32-a56a-fcc4998d383d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM1Ni0wMGU4ZTNmYi00MTRjLTRmMzItYTU2YS1mY2M0OTk4ZDM4M2QucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjUxMzczOGY0NjlmY2IyNTg0MWU4NzQ4NDBjMWUwZjdkMWViM2I3OWVhZTY2NDg1M2Q1YjczMzVjZjFkMWQxMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.3lqja1hgKl2qRfvjVJ0iVLlcSerXNceMELu1SFVMo3o)
> 
> Claiming:
> 
> ![Screenshot from 2024-09-12 09-33-59](https://private-user-images.githubusercontent.com/1193222/366769358-4898c775-25c0-45da-b760-a01e6040de90.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM1OC00ODk4Yzc3NS0yNWMwLTQ1ZGEtYjc2MC1hMDFlNjA0MGRlOTAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmJiODg2MDM0NzI4NGQwYzU1MWEyNDViM2I3NjIyMzBmYjViOWFiODM5ZmFjZTA1ZjVhMTVmNDI1OGYyYmM1NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Zj6LE57_vP1jmeuNkSTqGn5ldlDOVsi6P0pQXL2NLdw) ![Screenshot from 2024-09-12 09-34-25](https://private-user-images.githubusercontent.com/1193222/366769360-8ac8adca-e453-4edb-b42e-1720526a6df0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM2MC04YWM4YWRjYS1lNDUzLTRlZGItYjQyZS0xNzIwNTI2YTZkZjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGFiMDIzZjM0ODU1ZWIyODkyNjUwOGRjNTEzNzJlZGMwYmQ0ZDlhOTlhZDk1ZTA2NWU3ZjE5YjViYzNlMzQ4NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FAgPkNtqIbBy0NepGZA7IgUxMPf98bQUxSBwgYeMy0w)
> 
> Deposit Modal:
> 
> ![Screenshot from 2024-09-12 09-34-51](https://private-user-images.githubusercontent.com/1193222/366769364-c18b443c-77d1-46c5-8488-36a4d3090fff.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM2NC1jMThiNDQzYy03N2QxLTQ2YzUtODQ4OC0zNmE0ZDMwOTBmZmYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NzkzYmIwY2ZkZGEyOTUyYzI0MDQ1OGJjNDg0NTA3MjdhZDRjOTg5M2Y3YTQ2MDdmYjliYzk3YjhlZGI2YzA5YiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.eDEUp0xp-QgfwSL7M4iL-EIdFmRQpntwsToIN7kdd1c)
> 
> Simple Payment pre-filled:
> 
> ![Screenshot from 2024-09-12 09-35-19](https://private-user-images.githubusercontent.com/1193222/366769368-57f359f9-eb93-4262-a121-bab7d31f6b2a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM2OC01N2YzNTlmOS1lYjkzLTQyNjItYTEyMS1iYWI3ZDMxZjZiMmEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NjJmZGY1NWRjNDRiNjY3ZjMxYjhlZDY2NzBmZGI2OTIzMzc0NTQwNmI0ODA1YTQ3MjJkYjBiYjBkNTM4ZGYyNyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.Q45fd1UkE-zCyRWs4jK-FjGa4Am04y7OOXPCTrGBnkE)
> 
> As you can see when "Funds claimed" success message shows up, if the amount is large enoung _(well... long enough anyway)_, it will push the button(s) the next row, causing the UI to break slightly.
> 
> ![Screenshot from 2024-09-12 09-34-25](https://private-user-images.githubusercontent.com/1193222/366769360-8ac8adca-e453-4edb-b42e-1720526a6df0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM2MC04YWM4YWRjYS1lNDUzLTRlZGItYjQyZS0xNzIwNTI2YTZkZjAucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NGFiMDIzZjM0ODU1ZWIyODkyNjUwOGRjNTEzNzJlZGMwYmQ0ZDlhOTlhZDk1ZTA2NWU3ZjE5YjViYzNlMzQ4NiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FAgPkNtqIbBy0NepGZA7IgUxMPf98bQUxSBwgYeMy0w)
> 
> As a side note, and not related to this PR necessarily, I couldn't manage to apply your patch manually:
> 
> ![Screenshot from 2024-09-12 09-28-53](https://private-user-images.githubusercontent.com/1193222/366769343-71494076-e811-4abe-9c00-c7884c2524cb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM0My03MTQ5NDA3Ni1lODExLTRhYmUtOWMwMC1jNzg4NGMyNTI0Y2IucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDI2MmQ1ODQxYTM1ZjYwNTc4NTE0ZGJmYmI3OWZiZWIzZWYxZmIxNWI5OGVhOTg0NjkwN2JhNWE4NzY0MWVmNSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.gYw12hIp4ymG4VwOS920HAiCesaUStU165H-qVeT5MQ)
> 
> Which is super weird, since I made the changes myself, by hand, and then compared your patch and my changes, and, besides one empty new line, they're the same.
> 
> The empty line can't really be the cause since that is removed by default automatically anyway when you save the file.
> 
> ![Screenshot from 2024-09-12 09-32-33](https://private-user-images.githubusercontent.com/1193222/366769344-a20a56ce-4579-435c-afb6-075e56a1974f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MTAsIm5iZiI6MTcyNjIwNzIxMCwicGF0aCI6Ii8xMTkzMjIyLzM2Njc2OTM0NC1hMjBhNTZjZS00NTc5LTQzNWMtYWZiNi0wNzVlNTZhMTk3NGYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDkxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA5MTNUMDYwMDEwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGM5NDRiOGZmMzQxNDY3M2RlYTNmNTk3ZjViODdkOTNhNThmZTgwNjNlMDAxYzY2Y2ViOWE4NWQwM2RlZWRkZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.FyJ4qu27jeiIDJjiBDgRvZvbc0tVoxHqeM1oUIE0oGY)

Thank you for reviewing this @rdig 🙏 

I'll take care of the buttons breaking into the new line 💯 

Regarding the patch this is very weird. I assume it might have to do with the new line, but I'll try again later today

mmioana commented:
> > Regarding the modal background blur, I didn't change any behaviour regarding it, so if @melyndav considers this to be needed, we can create a separate issue for it 🌞
> 
> @mmioana Just a note to say this is blurred on master so something has broken along the way ~(If the issue hasn't been introduced in this PR, then I suspect this is a problem on the navigation branch, but I haven't checked it there, it probably makes more sense to fix it there if that is the case)~ Confirming this is an issue on `ui/new-navigation-ui` and it should be fixed there. I'll open an issue for it.
> 
> <img alt="Screenshot 2024-09-12 at 09 10 59" width="1728" src="https://private-user-images.githubusercontent.com/34915414/366782424-1cb04c5e-33ff-4d89-9678-e8351949418d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjYyMDc1MDksIm5iZiI6MTcyNjIwNzIwOSwicGF0aCI6Ii8zNDkxNTQxNC8zNjY3ODI0MjQtMWNiMDRjNWUtMzNmZi00ZDg5LTk2NzgtZTgzNTE5NDk0MThkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA5MTMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwOTEzVDA2MDAwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgwNTI2MzdiN2E2N2ZlN2RhYjYwOTQyMjFmYWYzYjNiZDI3N2EyYmI1ODZjZDUzMWJkMTQyNDE0YzM2OGFhMDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.oTHC0dwUdir-_7G__EW8SokFz8MYpSHEF8QWI3Qtlm8">

Thank you @iamsamgibbs for checking this 🙏 💯

iamsamgibbs reviewed (changes requested):
Nice work on this! I wasn't able to apply the diff either, but was otherwise able to follow the testing steps.

Income showing the correct value:

<img width="769" alt="Screenshot 2024-09-13 at 12 11 38" src="https://github.com/user-attachments/assets/e1b798f2-dfc2-4362-9d07-25125d1060f0">

Then after minting:

<img width="1139" alt="Screenshot 2024-09-13 at 12 13 39" src="https://github.com/user-attachments/assets/67cf3ce8-1cc1-49c8-b35d-b1db97068453">

Then after minting (without the auto claim)

<img width="743" alt="Screenshot 2024-09-13 at 12 19 24" src="https://github.com/user-attachments/assets/d6590ea6-6c65-4d23-bdbc-481095a7c1a4">

The funds claim button appeared correctly, but the value didn't refresh:

<img width="748" alt="Screenshot 2024-09-13 at 12 19 42" src="https://github.com/user-attachments/assets/6c1ffa20-1c1d-42c4-ae2c-cd47543a4594">

Only after a refresh did it update:

<img width="751" alt="Screenshot 2024-09-13 at 12 21 50" src="https://github.com/user-attachments/assets/ab240546-ca0c-43de-a115-759c1c5c2fb2">

This video shows it a little better:

https://github.com/user-attachments/assets/bf27dff8-628c-44ba-bd5b-abff66e6aa82

The modal and action form open correctly:
<img width="1387" alt="Screenshot 2024-09-13 at 12 29 43" src="https://github.com/user-attachments/assets/60b665a2-7a11-43da-8a44-8369152eb22f">

<img width="1376" alt="Screenshot 2024-09-13 at 12 29 36" src="https://github.com/user-attachments/assets/e9991c44-1e2c-464d-8927-d5b1d9a4b83c">

I couldn't get any payments to show when the "All teams" filter was selected. (Note the payments to a member and another colony in the activity feed)

<img width="1130" alt="Screenshot 2024-09-13 at 12 35 24" src="https://github.com/user-attachments/assets/47252847-7faf-4725-baa2-8565a8ba263f">

Payments do show when "General" or any other team is selected though.

<img width="1136" alt="Screenshot 2024-09-13 at 13 25 04" src="https://github.com/user-attachments/assets/ef75b557-9d26-4e59-8e9c-c8be4387d168">

Payments appear to be a bit broken however, in this video I make a payment of 1,000 CREDS, but almost £2,000 is shown.

https://github.com/user-attachments/assets/30a38ec8-44f7-4582-844a-6d5e6390c348

I also noticed some differences in the loading skeleton:

<img width="713" alt="Screenshot 2024-09-13 at 12 57 15" src="https://github.com/user-attachments/assets/e216b623-0e47-449b-a2e9-eb6aac5f2bb0">

Figma:
<img width="861" alt="Screenshot 2024-09-13 at 12 57 52" src="https://github.com/user-attachments/assets/f907b3aa-8a7e-4c65-965d-dd1c202c8315">

I'm guessing the missing block is the trend arrows which are yet to be added - but some of the loaders appear more rounded in figma.

Appreciate some of this is likely out of scope for this issue! 😅

mmioana commented:
@rdig and @iamsamgibbs thank you a lot for your throughout review 🙏 

After discussing with @melyndav and @arrenv, the button shown after claiming funds will only display `Claimed` - without the tick icon - thus the buttons overflowing to the next row shouldn't happen anymore

Also, I have updated the patch file content - there might have been some spacing issues causing the `git apply <_filename_here_>` to not work as expected

@iamsamgibbs I have added a refetch call after claiming the funds. Thank you for spotting this 💯 
Regarding the payments value, there is another issue I have created after talking with @jakubcolony as there definitely is an overlap between the simple payment actions and expenditures, thus having a wrong value returned from the lambda - https://github.com/JoinColony/colonyCDapp/issues/3067

Also, I have increased the border-radius for the button's loading skeletons from `4px` to `8px`, as per Figma

Thank you for spotting these issues and please give it another go 🙏

iamsamgibbs reviewed (changes requested):
Thanks for making those changes! Tested and everything is still working as expected. The diff worked this time too thanks!

Rounded corners look better on the loading skeleton:

![Screenshot 2024-09-16 at 15 54 29](https://github.com/user-attachments/assets/b23418ae-cbc6-4734-b8a3-0ae0bee10632)

But the refetch doesn't seem to be happening after a claim, not sure if this is just me?

https://github.com/user-attachments/assets/7ad5f749-9b0f-4b8a-b751-098be08bf6b5

mmioana commented:
Thank you @iamsamgibbs for checking again! 
I have pushed now a different fix, so that both the Income/Payments values get updated and the bar chart, if you want to give it another go 🙏

iamsamgibbs reviewed (approved):
Nice job! The income is refetched properly now when claiming. All looks good to me!

https://github.com/user-attachments/assets/a36b3f3b-8f4c-431c-9a0a-f35ff6c67e4e

rdig reviewed (approved):
Re-approved

Nortsova reviewed (approved):
Thanks for changes! 🙌  Reapproved ✅

Value was updated after minting tokens:
<img width="568" alt="image" src="https://github.com/user-attachments/assets/a3965524-4d36-4641-9516-ba265cbde4af">
I was able to claim incoming funds:
![claim](https://github.com/user-attachments/assets/e0126a95-fb67-47d9-9450-10344f83a0e5)
After that claim button disappear
